utilities/cassandra/format.h

```diff
@@ -167,6 +167,7 @@ class RowValue {
   RowValue ConvertExpiredColumnsToTombstones(bool* changed) const;
   RowValue RemoveTombstones(int32_t gc_grace_period) const;
   bool Empty() const;
+  size_t ColumnSize() const;
```

Comment:
column size is not related to this PR also? seems not used.

Reply:
good catch